app/database/MetricsDB.scala

```diff
@@ -21,8 +23,10 @@ class MetricsDB(val db: Database) {
   def getForks: Seq[Fork] = await(db.run(forksTable.result))
   def insertFork(fork: Fork): Int = await(db.run(forksTable += fork))
 
+  val dayOfWeek = SimpleFunction.binary[String, Timestamp, Timestamp]("date_trunc")
```

Comment:
As far as I can tell, it returns the desired result http://www.postgresqltutorial.com/postgresql-date_trunc/